cloud/blockstore/tools/csi_driver/internal/driver/node_test.go

```diff
@@ -713,15 +711,11 @@ func TestGetVolumeStatCapabilitiesWithoutVmMode(t *testing.T) {
 }
 
 func TestGetVolumeStatCapabilitiesWithVmMode(t *testing.T) {
```

Comment:
One more question: now we support only publish path, but comment in [spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#nodegetvolumestats) seems to allow staging or publish path to be an argument for NodeGetVolumeStats. So looks like this call will fail if staging path is requested.
Am I wrong or should I create an issue for that?

Reply:
Yep, it's a bug.

In our CSI Driver implementation we completely ignore staging path however it is required to mount volume to staging directory too.

Reply:
should be fixed as part of https://nebius.atlassian.net/browse/NBSNEBIUS-401